src/components/StepTitle/index.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import React, { ReactNode } from 'react'
+import Bullet from '../Bullet'
+
+export const Step = styled('div', {
+  shouldForwardProp: prop => !['disabled'].includes(prop.toString()),
+})<{ disabled: boolean }>`
+  margin-bottom: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: ${({ theme, disabled }) =>
+    disabled ? theme.colors.neutral.textWeak : theme.colors.neutral.textStrong};
+`
+
+const StyledDiv = styled('div', {
+  shouldForwardProp: prop => !['size'].includes(prop.toString()),
+})<{ size: Sizes }>`
+  flex: 1;
+  margin-left: ${({ theme, size }) =>
+    size === 'medium' ? theme.space['2'] : theme.space['1']};
+  font-size: ${({ size }) => (size === 'medium' ? '24px' : '16px')};
+`
+
+export type Sizes = 'small' | 'medium' | undefined
+
+interface StepTitleProps {
+  bulletText: string
+  size?: Sizes
+  disabled?: boolean
+  keyPrefix?: string
+  children: ReactNode
+}
+
+const StepTitle = ({
+  bulletText,
+  children,
+  keyPrefix,
+  size = 'medium',
+  disabled = false,
+}: StepTitleProps) => (
+  <Step
+    key={`${keyPrefix ? `${keyPrefix}-` : ''}step-${bulletText}`}
+    disabled={disabled}
+  >
+    <Bullet
+      text={bulletText}
+      size={size}
+      variant={disabled ? 'disabled' : 'default'}
+    />
+    <StyledDiv size={size}>{children}</StyledDiv>
+  </Step>
+)
```

Comment:
I'm pretty sure we will need to customize bullet icon and variant too. For example to do a successful stepTitle.

Maybe we should add it into props too, what do you think ?

Reply:
Done & RTR @matthprost